utils/test_move_and_process.py

```diff
@@ -62,12 +74,13 @@ def test_generate(self):
             self.assertEqual(out_fns, [c[-1] for c in cmds])
 
     def test_get_loudness(self):
-        input = '/tmp/original/test.ogv'
-        integrated_lufs, truepeak_lufs = mp.get_loudness(input)
-        self.assertFalse(None == integrated_lufs)
-        input = '/tmp/large_thumb/test.jpg'
-        integrated_lufs, truepeak_lufs = mp.get_loudness(input)
-        self.assertTrue(None == integrated_lufs)
+        loudness  = mp.get_loudness(self.audiofile)
+        print(loudness)
+        self.assertTrue(-2.2 == loudness['integrated_lufs']
+                         and 0.54 == loudness['truepeak_lufs'])
+        loudness = mp.get_loudness(self.imagefile)
+        print(loudness)
+        self.assertTrue({} == loudness)
```

Comment:
If you want to have more than one test in a file, I think you should make every "test" _one_ entry. You can i.e. do that like this:

```
self.assertEqual(mp.get_loudness('sound.wav'), {integrated_lufs: -2.2, truepeak_lufs: 0.54})
self.assertEqual(mp.get_loudness('image.jpg'), {})
```